.travis.yml

```
@@ -1,28 +1,24 @@
 # Travis Continuous Integration Configuration File For ROS Control Projects
 # Author: Dave Coleman
-sudo: required
-dist: trusty
 language: generic
+services:
+  - docker
 
 notifications:
   email:
     recipients:
-      - enrique.fernandez.perdomo@gmail.com
       - bence.magyar.robotics@gmail.com
-      - gennaro.raiola@gmail.com
     on_success: change #[always|never|change] # default: change
     on_failure: change #[always|never|change] # default: always
 
 env:
   global:
-    - UPSTREAM_WORKSPACE=file
-    - ROSINSTALL_FILENAME=ros_control.rosinstall
-    - ROS_PARALLEL_TEST_JOBS=-j1
+    - UPSTREAM_WORKSPACE=ros_control.rosinstall
```

```suggestion
    - UPSTREAM_WORKSPACE='ros_control.rosinstall -ros_control'
```
To prevent from buliding `ros_control` twice.

---

thanks!